docs/index.mdx

```diff
@@ -58,6 +58,12 @@ The following projects are using Melos:
 - [GrowERP Flutter ERP](https://github.com/growerp/growerp)
 - [mrverdant13/coverde](https://github.com/mrverdant13/coverde)
 - [woltapp/wolt_modal_sheet](https://github.com/woltapp/wolt_modal_sheet)
+- [cfug/dio](https://github.com/cfug/dio)
+- [simolus3/drift](https://github.com/simolus3/drift)
+- [Lyokone/flutterlocation](https://github.com/Lyokone/flutterlocation)
+- [FlutterGen/flutter_gen](https://github.com/FlutterGen/flutter_gen)
+- [canonical/ubuntu-desktop-installer](https://github.com/canonical/ubuntu-desktop-installer)
```

Comment:
This one is being deprecated, it's better to add the new one instead:
```suggestion
- [canonical/ubuntu-desktop-provision](https://github.com/canonical/ubuntu-desktop-provision)
```

Reply:
Let's do so, replaced with the new one!